packages/date-picker/src/vaadin-infinite-scroller.js

```diff
@@ -155,28 +144,52 @@ class InfiniteScroller extends PolymerElement {
     }
   }
 
+  /**
+   * @protected
+   * @override
+   */
+  _createElement() {
+    // To be implemented.
+  }
+
+  /**
+   * @param {HTMLElement} _element
+   * @param {number} _index
+   * @protected
+   * @override
+   */
+  _updateElement(_element, _index) {
+    // To be implemented.
+  }
+
+  /** @private */
   _activated(active) {
     if (active && !this._initialized) {
       this._createPool();
       this._initialized = true;
     }
   }
 
+  /** @private */
   _finishInit() {
     if (!this._initDone) {
       // Once the first set of items start fading in, stamp the rest
       this._buffers.forEach((buffer) => {
-        [].forEach.call(buffer.children, (insertionPoint) => this._ensureStampedInstance(insertionPoint._itemWrapper));
+        [...buffer.children].forEach((slot) => {
+          this._ensureStampedInstance(slot._itemWrapper);
+        });
       });
 
       if (!this._buffers[0].translateY) {
         this._reset();
       }
 
       this._initDone = true;
+      this.dispatchEvent(new CustomEvent('init-done'));
```

Comment:
The event is used to notify overlay content so that we can get reference to calendars and years.
Could be probably refactored to use `afterNextRender` in the overlay content instead.